pages/personal.tsx

```diff
@@ -1,38 +1,81 @@
 import dynamic from 'next/dynamic'
-import ProfileImage from '../components/ProfileImage'
+import Head from 'next/head'
 import { NavBar } from '../components/NavBar'
+import ProfileImage from '../components/ProfileImage'
 import { PersonalDescription } from '../components/PersonalDescription'
-import { Container, Center } from '@chakra-ui/react'
-import { useState } from 'react'
-import { CommentSection } from '../components/CommentSection'
+import { Box, SimpleGrid, VStack } from '@chakra-ui/react'
+import { useState, useEffect } from 'react'
 
 const LazyVisualizer = dynamic(() => import('../components/AudioVisualizer'), {
   ssr: false
 })
 
 const personal = () => {
-  interface Data {
-    imgURL: string,
-    name: string,
-    instruments: string[],
-  }
+
+  useEffect(() => {
+    fetch('api/userFeed')
+      .then(async (response) => {
+        const newData = await response.json()
+        setData(newData)
+      })
+  },[])
 
   const [data, setData] = useState({
-    imgURL: 'https://newprofilepic2.photo-cdn.net//assets/images/article/profile.jpg',
-    name: 'Tracy Hillberg',
-    instruments: ['cello', 'flute', 'drums'],
-  })
+    "name": "Ivan",
+    "description": "hello world",
+    "instruments": [
+        "Cello",
+        "Piano",
+        "Drums"
+    ],
+    "image": "testprofilepicture.jpg",
+    "posts": [
+        {
+            "name": "Joe",
+            "band": "Super Sick Band",
+            "audio": "testaudio.wav",
+            "pdf": "testpdf.pdf",
+            "date": "01/17/2023 @ 8:09pm",
+            "text": "Hello user feed",
+            "comments": [
+                {
+                    "name": "Darrien",
+                    "profile_picture": "sampleprofpic.jpg",
+                    "text": "hello comments",
+                    "date": "01/17/2023 @ 8:10pm"
+                },
+                {
+                    "name": 'Joe',
+                    "profile_picture": 'pfp.jpeg',
+                    "text": "test",
+                    "date": '01/17/2023 @ 8:11pm'
+                }
+            ]
+        }
+    ]
+})
 
 
   return(
     <>
-      <NavBar />
-      <Container>
-        <ProfileImage imgURL={data.imgURL} name={data.name}/>
-        <PersonalDescription instruments={data.instruments} sectionName='personal'/>
-        <LazyVisualizer/>
-        <CommentSection/>
-      </Container>
+    <Head>
+      <title>Your Homepage</title>
+    </Head>
+
+      <Box h='100vh' w='100vw'>
```

Comment:
I'm still getting a horizontal scroll on the personal page 😅
I think adding `maxW='100%'` might fix this? Let me know how it looks on your end.

Reply:
Changing that makes no difference for me so I dont know if it will work on your end or not, try it on your local machine and let me know if it works